src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "CSS backdrop-filter"
```

Comment:
i agree the title could be better, but maybe it could be something like "CSS backdrop-filter official release"? this is the general response from folks, since the css feature is 3 years old or so and has been mildly usable all that time. i'm struggling with titles that dont compete with the subtitle. and i feel like the subtitle is the better place to expand upon the concept. 

maybe a title more like "Chrome Lands Stable CSS backdrop-filter"? or "Chrome 76 Stabilizes CSS backdrop-filter"?

🤔

Reply:
I'd suggest steering away from calling it an official release because other folks could point out that it's been released in other browsers for quite some time so it might sound presumptuous for Chrome to decide when something is "officially" released.

We really want web.dev to feel browser agnostic so I think highlighting what the feature gives folks (translucent effects in CSS) without singling out any one browser in the title would be good. Thomas' post on dark mode is a good example of this: https://web.dev/prefers-color-scheme/

It highlights the feature and then uses the post as an opportunity to get the topic on people's radars again and list the browser support (which now includes Chrome). I think a good way to do that is to include a version number when you mention Chrome and the other browser. For example, "This is supported in Chrome 76, Safari, Firefox, ..."